test-vectors/message.json

```diff
@@ -0,0 +1,32 @@
+{
+  "class": "message",
+  "selector": "<some predicate for the driver to determine if this vector applies to the implementation, e.g. to deal with protocol upgrades>",
+
+  "_meta": {
+    "id": "test_vector_id",
+    "version": "version_id",
+    "gen": {
+      "source": "lotus",
+      "version": "0.4.1+git.27d74337+api0.8.1"
+    }
+  },
+
+  "preconditions": {
+    "state_tree": {
+      "t01": "0x0123456789abcdef",
+      "t02": "0x0123456789abcdef",
+      "t03": "0x0123456789abcdef"
+    }
+  },
+
+  "apply_messages": [
```

Comment:
Depending on the test vector class, the JSON schema will enforce one of the following fields:

* `apply_message`
* `apply_block`
* `apply_tipset`
* `apply_chain`